fiftyone/server/routes/sample.py

```
@@ -645,10 +645,118 @@ async def patch(self, request: Request, data: List[dict]) -> JSONResponse:
+                detail="Both 'field' and 'detection_id' are required",
+            )
+
+        sample = get_sample(dataset_id, sample_id, if_last_modified_at)
+
+        try:
+            label_field = sample[field]
+        except KeyError:
+            raise HTTPException(
+                status_code=404,
+                detail=f"Field '{field}' not found on sample",
+            )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)